cmd/check_container.go

```diff
@@ -152,6 +152,8 @@ var checkContainerCmd = &cobra.Command{
 			return err
 		}
 
+		log.Infof("Preflight result: %s", covertPassedOverall(results.PassedOverall))
```

Comment:
I think this needs to be at the very end. So that it's the absolute last thing logged.